.github/workflows/ci.yml

```
@@ -2,167 +2,132 @@ name: CI
-        PYTHONPATH: src
-      run: |
-        python -m py_compile src/madspark/core/coordinator.py
-        python -m py_compile src/madspark/agents/*.py
+        echo "Running basic Python syntax checks..."
+        find . -name "*.py" -type f -exec python -m py_compile {} \;
+        echo "✅ Python syntax check passed"
+
+  security-scan:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)